add `hlt` (`HLT`) label to Phase-2 HLT modules (sequences)

mmusich opened the pull request:
#### PR description:

The goal of this PR is to package together the renaming of the HLT module and sequences labels in the HLT phase-2 menu (kindly provided by @AuroraPerego in https://github.com/AuroraPerego/cmssw/commit/226b557a98de14281abb5eff45a56250ffd61f66, see discussion at https://github.com/cms-sw/cmssw/pull/45500#issuecomment-2242744894) with the necessary updates of the Trigger DQM and Validation sequences configurations consuming said products in order to stay in synch with it (that I provide in https://github.com/mmusich/cmssw/commit/24f475671b40fa6a7fc7c6e91abcd98c3d0fa09d)

#### PR validation:

None, will rely on the bot tests.

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

This PR is not a backport and it should not be backported.

***

I open this PR in draft mode as I am not sure that I have caught  all the products consumed downstream that need to be renamed in the configuration.
I would like to request feedback to:
  * @cms-sw/tracking-pog-l2 @elusian @lguzzi @cms-sw/trk-dpg-l2  about the new naming of the collections used for tracker local reconstruction / tracking / vertexing
  * @cms-sw/pf-l2 @waredjeb @jsamudio about the naming of the PF modules used in the HLT.
  * @cms-sw/hgcal-dpg-l2 if there is any DQM / validation module monitoring HGCal products that need to be taken care of.  

Cc: @rovere @SohamBhattacharya @VourMa

AuroraPerego committed:
add hlt label to Phase-2 HLT modules and sequences

- add `hlt` to modules
- add `HLT` to sequences

mmusich committed:
rename input collections for tracking @ HLT monitoring for Phase-2

cmsbuild commented:
cms-bot internal usage<!-- bot cache: {"commits":{"24f475671b40fa6a7fc7c6e91abcd98c3d0fa09d":{"files":["DQMOffline/Trigger/python/PrimaryVertexMonitoring_cff.py","DQMOffline/Trigger/python/SiTrackerPhase2_OfflineMonitoring_cff.py","DQMOffline/Trigger/python/TrackToTrackMonitoring_cff.py","DQMOffline/Trigger/python/TrackingMonitoring_cff.py","Validation/RecoTrack/python/HLTmultiTrackValidator_cff.py","Validation/RecoTrack/python/associators_cff.py","Validation/RecoVertex/python/HLTmultiPVvalidator_cff.py"],"squashed":false,"time":1722842895},"f2bce8075e6132202aacb8e94a44063f38cd8575":{"files":["HLTrigger/Configuration/python/HLT_75e33/modules/ecalBarrelClusterFastTimer_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/fixedGridRhoFastjetAllTmp_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hgcalMergeLayerClustersL1Seeded_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hgcalMergeLayerClusters_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/highPtTripletStepTrackSelectionHighPurity_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltAK4PFCHSJetCorrectorL1_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltAK4PFCHSJets_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltAK4PFClusterJets_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltAK4PFJetCorrectorL1_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltAK4PFJetsForTaus_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltAK4PFJets_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltAK4PFPuppiJetCorrectorL1_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltAK4PFPuppiJets_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltAK8PFCHSJetCorrectorL1_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltAK8PFCHSJets_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltAK8PFClusterJets_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltAK8PFJetCorrectorL1_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltAK8PFJets_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltAK8PFPuppiJetCorrectorL1_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltAK8PFPuppiJets_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltAk4CaloJetsForTrk_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltBunchSpacingProducer_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltCaloMET_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltDeepBLifetimeTagInfosPFPuppiModEta2p4_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltDeepBLifetimeTagInfosPFPuppi_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltDeepInclusiveVertexFinderPF_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltDeepTrackVertexArbitratorPF_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltEcalBarrelClusterFastTimer_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltEcalDetailedTimeRecHit_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltEcalMultiFitUncalibRecHit_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltEcalRecHit_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltEgammaCandidatesL1Seeded_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltEgammaCandidatesUnseeded_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltEgammaCkfTrackCandidatesForGSFL1Seeded_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltEgammaCkfTrackCandidatesForGSFUnseeded_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltEgammaEleGsfTrackIsoL1Seeded_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltEgammaEleGsfTrackIsoUnseeded_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltEgammaEleGsfTrackIsoV6L1Seeded_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltEgammaEleGsfTrackIsoV6Unseeded_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltEgammaElectronPixelSeedsL1Seeded_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltEgammaElectronPixelSeedsUnseeded_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltEgammaGsfTracksL1Seeded_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltEgammaGsfTracksUnseeded_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltEgammaHGCALIDVarsL1Seeded_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltEgammaHGCALIDVarsUnseeded_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltEgammaHGCalLayerClusterIsoL1Seeded_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltEgammaHGCalLayerClusterIsoUnseeded_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltEgammaHollowTrackIsoL1Seeded_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltEgammaHollowTrackIsoUnseeded_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltElePixelHitTripletsClusterRemoverL1Seeded_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltElePixelHitTripletsClusterRemoverUnseeded_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltFilteredLayerClustersCLUE3DHighL1Seeded_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltFilteredLayerClustersCLUE3DHigh_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltFilteredLayerClustersPassThrough_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltFirstStepPrimaryVerticesUnsorted_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltFixedGridRhoFastjetAllTmp_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltFixedGridRhoProducerFastjetAllTau_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltGeneralTracks_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltGoodOfflinePrimaryVertices_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltHGCalRecHitL1Seeded_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltHGCalRecHit_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltHGCalUncalibRecHitL1Seeded_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltHGCalUncalibRecHit_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltHgcalDigisL1Seeded_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltHgcalDigis_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltHgcalLayerClustersEEL1Seeded_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltHgcalLayerClustersEE_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltHgcalLayerClustersHSciL1Seeded_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltHgcalLayerClustersHSci_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltHgcalLayerClustersHSiL1Seeded_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltHgcalLayerClustersHSi_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltHgcalMergeLayerClustersL1Seeded_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltHgcalMergeLayerClusters_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltHgcalSoARecHitsProducer_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltHgcalTrackCollection_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltHighPtTripletStepClusters_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltHighPtTripletStepHitDoublets_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltHighPtTripletStepHitTriplets_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltHighPtTripletStepSeedLayers_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltHighPtTripletStepSeeds_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltHighPtTripletStepTrackCandidates_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltHighPtTripletStepTrackCutClassifier_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltHighPtTripletStepTrackSelectionHighPurity_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltHighPtTripletStepTracks_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltHpsCombinatoricRecoTaus_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltHpsPFTauBasicDiscriminatorsForDeepTau_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltHpsPFTauBasicDiscriminatorsdR03ForDeepTau_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltHpsPFTauDeepTauProducer_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltHpsPFTauMediumAbsoluteChargedIsolationDiscriminator_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltHpsPFTauMediumRelativeChargedIsolationDiscriminator_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltInitialStepSeeds_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltInitialStepTrackCandidates_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltInitialStepTrackCutClassifier_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltInitialStepTrackSelectionHighPurity_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltInitialStepTracks_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltIter0Phase2L3FromL1TkMuonCkfTrackCandidates_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltIter0Phase2L3FromL1TkMuonCtfWithMaterialTracks_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltIter2Phase2L3FromL1TkMuonClustersRefRemoval_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltIter2Phase2L3FromL1TkMuonMaskedMeasurementTrackerEvent_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltIter2Phase2L3FromL1TkMuonPixelClusterCheck_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltIter2Phase2L3FromL1TkMuonPixelLayerTriplets_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltMeasurementTrackerEvent_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltMuonCSCDigis_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltMuonDTDigis_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltMuonGEMDigis_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltOfflinePrimaryVertices_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltPFClusterMET_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltPFMETJetCorrectorL1_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltPFMET_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltPFPuppiMET_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltPFPuppiNoLep_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltPFPuppi_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltParticleFlowBadHcalPseudoCluster_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltParticleFlowBlock_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltParticleFlowClusterECALUncorrected_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltParticleFlowClusterECAL_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltParticleFlowClusterHF_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltParticleFlowClusterHGCalFromTICLL1Seeded_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltParticleFlowClusterHGCalFromTICLUnseeded_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltParticleFlowClusterHGCal_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltParticleFlowClusterHO_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltParticleFlowPtrs_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltParticleFlowRecHitECAL_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltParticleFlowRecHitHF_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltParticleFlowRecHitHGCL1Seeded_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltParticleFlowRecHitHGC_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltParticleFlowRecHitHO_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltParticleFlowSuperClusterECALL1Seeded_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltParticleFlowSuperClusterECALUnseeded_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltParticleFlowSuperClusterECAL_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltParticleFlowSuperClusterHGCalFromTICLL1Seeded_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltParticleFlowSuperClusterHGCalFromTICLUnseeded_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltParticleFlowSuperClusterHGCal_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltParticleFlowTimeAssignerECAL_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltParticleFlowTmpBarrel_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltParticleFlowTmp_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltPfClusterRefsForJetsECAL_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltPfClusterRefsForJetsHCAL_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltPfClusterRefsForJetsHF_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltPfClusterRefsForJetsHGCAL_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltPfClusterRefsForJetsHO_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltPfClusterRefsForJets_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltPfDeepFlavourTagInfosModEta2p4_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltPfDeepFlavourTagInfos_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltPfNoPileUpJME_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltPfPileUpJME_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltPfTICL_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltPfTrack_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltPhase2L3FromL1TkMuonPixelLayerQuadruplets_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltPhase2L3FromL1TkMuonPixelTracksHitQuadruplets_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltPhase2L3MuonHighPtTripletStepClusters_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltPhase2L3MuonHighPtTripletStepHitDoublets_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltPhase2L3MuonHighPtTripletStepHitTriplets_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltPhase2L3MuonHighPtTripletStepSeedLayers_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltPhase2L3MuonHighPtTripletStepTrackCandidates_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltPhase2L3MuonHighPtTripletStepTracks_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltPhase2L3MuonInitialStepTrackCandidates_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltPhase2L3MuonInitialStepTracks_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltPhase2L3MuonPixelTracksHitDoublets_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltPhase2L3MuonPixelTracksHitQuadruplets_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltPhase2L3MuonPixelTracksSeedLayers_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltPhase2L3MuonsHgcalLCIsodR0p2dRVetoEM0p00dRVetoHad0p02minEEM0p00minEHad0p00_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltPhase2L3OIMuCtfWithMaterialTracks_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltPhase2L3OISeedsFromL2Muons_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltPhase2L3OITrackCandidates_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltPhase2PixelTracksHitSeeds_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltPhase2PixelTracksSeedLayers_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltPixelClustersMultiplicity_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltPixelLayerPairsL1Seeded_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltPixelLayerPairsUnseeded_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltPixelLayerTriplets_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltPrimaryVertexAssociationModEta2p4_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltPrimaryVertexAssociation_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltRechitInRegionsHGCAL_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltSiPhase2Clusters_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltSiPixelClusterShapeCache_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltSiPixelClusters_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltSiPixelRecHits_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltTauPFJets08Region_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltTiclCandidate_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltTiclLayerTileProducerL1Seeded_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltTiclLayerTileProducer_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltTiclSeedingGlobal_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltTiclSeedingL1_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltTiclTracksterLinks_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltTiclTrackstersCLUE3DHighL1Seeded_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltTiclTrackstersCLUE3DHigh_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltTiclTrackstersMerge_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltTiclTrackstersPassthrough_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltTowerMaker_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltTrackRefsForJetsBeforeSorting_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltTrackWithVertexRefSelectorBeforeSorting_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltTrackerClusterCheck_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/hltUnsortedOfflinePrimaryVertices_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/initialStepTrackSelectionHighPurity_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/particleFlowPtrs_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/particleFlowTimeAssignerECAL_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/particleFlowTmp_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/pfClusterRefsForJetsECAL_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/pfClusterRefsForJetsHF_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/pfClusterRefsForJetsHGCAL_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/pfClusterRefsForJetsHO_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/pfClusterRefsForJets_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/pfNoPileUpJME_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/siPixelClusterShapeCache_cfi.py","HLTrigger/Configuration/python/HLT_75e33/modules/trackRefsForJetsBeforeSorting_cfi.py","HLTrigger/Configuration/python/HLT_75e33/paths/HLT_AK4PFPuppiJet520_cfi.py","HLTrigger/Configuration/python/HLT_75e33/paths/HLT_DoubleMediumChargedIsoPFTauHPS40_eta2p1_cfi.py","HLTrigger/Configuration/python/HLT_75e33/paths/HLT_DoubleMediumDeepTauPFTauHPS35_eta2p1_cfi.py","HLTrigger/Configuration/python/HLT_75e33/paths/HLT_DoublePFPuppiJets128_DoublePFPuppiBTagDeepCSV_2p4_cfi.py","HLTrigger/Configuration/python/HLT_75e33/paths/HLT_DoublePFPuppiJets128_DoublePFPuppiBTagDeepFlavour_2p4_cfi.py","HLTrigger/Configuration/python/HLT_75e33/paths/HLT_IsoMu24_FromL1TkMuon_cfi.py","HLTrigger/Configuration/python/HLT_75e33/paths/HLT_Mu17_TrkIsoVVL_Mu8_TrkIsoVVL_DZ_FromL1TkMuon_cfi.py","HLTrigger/Configuration/python/HLT_75e33/paths/HLT_Mu37_Mu27_FromL1TkMuon_cfi.py","HLTrigger/Configuration/python/HLT_75e33/paths/HLT_Mu50_FromL1TkMuon_cfi.py","HLTrigger/Configuration/python/HLT_75e33/paths/HLT_PFHT200PT30_QuadPFPuppiJet_70_40_30_30_TriplePFPuppiBTagDeepCSV_2p4_cfi.py","HLTrigger/Configuration/python/HLT_75e33/paths/HLT_PFHT200PT30_QuadPFPuppiJet_70_40_30_30_TriplePFPuppiBTagDeepFlavour_2p4_cfi.py","HLTrigger/Configuration/python/HLT_75e33/paths/HLT_PFHT330PT30_QuadPFPuppiJet_75_60_45_40_TriplePFPuppiBTagDeepCSV_2p4_cfi.py","HLTrigger/Configuration/python/HLT_75e33/paths/HLT_PFHT330PT30_QuadPFPuppiJet_75_60_45_40_TriplePFPuppiBTagDeepFlavour_2p4_cfi.py","HLTrigger/Configuration/python/HLT_75e33/paths/HLT_PFPuppiHT1070_cfi.py","HLTrigger/Configuration/python/HLT_75e33/paths/HLT_PFPuppiMETTypeOne140_PFPuppiMHT140_cfi.py","HLTrigger/Configuration/python/HLT_75e33/paths/HLT_TriMu_10_5_5_DZ_FromL1TkMuon_cfi.py","HLTrigger/Configuration/python/HLT_75e33/paths/MC_BTV_cfi.py","HLTrigger/Configuration/python/HLT_75e33/paths/MC_JME_cfi.py","HLTrigger/Configuration/python/HLT_75e33/psets/ClusterShapeTrajectoryFilter_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/HLTAK4PFPuppiJetsReconstruction_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/HLTCaloTowersRecSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/HLTCalolocalrecoSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/HLTCsclocalrecoSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/HLTDoFullUnpackingEgammaEcalL1SeededSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/HLTDoFullUnpackingEgammaEcalSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/HLTDoLocalPixelSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/HLTDoLocalStripSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/HLTDtlocalrecoSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/HLTEcalDigisSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/HLTElePixelMatchL1SeededSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/HLTElePixelMatchUnseededSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/HLTEndOfProcess_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/HLTFastJetForEgammaSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/HLTGemLocalRecoSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/HLTHgcalLocalRecoSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/HLTHgcalTiclPFClusteringForEgammaL1SeededSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/HLTHgcalTiclPFClusteringForEgammaUnseededSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/HLTHgcalTiclPFClusteringForEgamma_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/HLTHighPtTripletStepSeedingSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/HLTHighPtTripletStepSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/HLTInitialStepPVSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/HLTInitialStepSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/HLTItLocalRecoSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/HLTIterTICLSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/HLTL2MuonsFromL1TkSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/HLTLocalrecoSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/HLTMuonlocalrecoSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/HLTOtLocalRecoSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/HLTPFClusterJMEReconstruction_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/HLTPFJetsCHSReconstruction_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/HLTPFPuppiMETReconstruction_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/HLTParticleFlowClusterECALSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/HLTParticleFlowClusterSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/HLTParticleFlowRecoSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/HLTParticleFlowSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/HLTParticleFlowSuperClusteringSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/HLTPfClusterRefsForJetsSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/HLTPfClusteringECALSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/HLTPfClusteringHBHEHFSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/HLTPfClusteringHOSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/HLTPhase2L3MuonGeneralTracksSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/HLTPhase2PixelTracksSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/HLTRawToDigiSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/HLTTiclLayerTileSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/HLTTiclPFSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/HLTTiclTracksterMergeSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/HLTTiclTrackstersCLUE3DHighStepSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/HLTTrackingV61Sequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/HLTVertexRecoSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/RawToDigiSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/caloTowersRecSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/hgcalLocalRecoSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/highPtTripletStepSeedingSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/highPtTripletStepSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/initialStepPVSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/initialStepSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/itLocalRecoSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/iterTICLSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/localrecoSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/muonlocalrecoSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/otLocalRecoSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/particleFlowClusterECALSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/particleFlowClusterSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/particleFlowRecoSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/particleFlowSuperClusteringSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/pfClusterRefsForJetsSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/pfClusteringECALSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/pfClusteringHBHEHFSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/pfClusteringHOSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/ticlLayerTileSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/ticlPFSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/ticlTracksterMergeSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/ticlTrackstersCLUE3DHighStepSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33/sequences/vertexRecoSequence_cfi.py","HLTrigger/Configuration/python/HLT_75e33_cff.py"],"squashed":false,"time":1722838223}},"emoji":{},"last_seen_sha":"24f475671b40fa6a7fc7c6e91abcd98c3d0fa09d","signatures":{}} -->

cmsbuild commented:
A new Pull Request was created by  @mmusich for master.

It involves the following packages:

- DQMOffline/Trigger (**dqm**)
- HLTrigger/Configuration (**hlt**)
- Validation/RecoTrack (**dqm**)
- Validation/RecoVertex (**dqm**)


@Martin-Grunewald, @antoniovagnerini, @cmsbuild, @mmusich, @nothingface0, @rvenditti, @syuvivida, @tjavaid can you please review it and eventually sign? Thanks.
@Fedespring, @GiacomoSguazzoni, @HuguesBrun, @JanFSchulte, @Martin-Grunewald, @ReyerBand, @SohamBhattacharya, @VinInn, @VourMa, @argiro, @cericeci, @dgulhan, @felicepantaleo, @jhgoh, @missirol, @mmusich, @mtosi, @rchatter, @rociovilar, @rovere, @silviodonato, @thomreis, @trocino, @venturia, @wang0jin, @wmtford this is something you requested to watch as well.
@antoniovilela, @mandrenguyen, @rappoccio, @sextonkennedy you are the release manager for this.

cms-bot commands are listed <a href="http://cms-sw.github.io/cms-bot-cmssw-cmds.html">here</a>

mmusich commented:
@cmsbuild, please test

AuroraPerego committed:
save TICL collections at HLT

waredjeb commented:
Hi @mmusich  thanks a lot! 
I think I might have to go through it couple of times! 

For what concerns TICL/HGCAL, I think we need to also modify the RecoHGCal Event content https://github.com/cms-sw/cmssw/blob/master/RecoHGCal/Configuration/python/RecoHGCal_EventContent_cff.py adding the new HLT collections to `FEVTDEBUGHLToutput`

We can make a commit on top of this PR such that you can just cherry-pick it

mmusich commented:
> I think we also need to modify the RecoHGCal Event content .

Good point! The same holds true for the tracking products .

cmsbuild commented:
+1

**Size**: This PR adds an extra 220KB to repository
**Summary**: https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-ea66c8/40760/summary.html
**COMMIT**: 24f475671b40fa6a7fc7c6e91abcd98c3d0fa09d
**CMSSW**: CMSSW_14_1_X_2024-08-04-2300/el8_amd64_gcc12
**User test area**: For local testing, you can use `/cvmfs/cms-ci.cern.ch/week1/cms-sw/cmssw/45636/40760/install.sh` to create a dev area with all the needed externals and cmssw changes.


## Comparison Summary



**Summary**:
 - You potentially [added 305 lines](https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_14_1_X_2024-08-04-2300+ea66c8/63879/validateJR/logRootQA-events.log) to the logs
 - ROOTFileChecks: Some differences in event products or their sizes found
 - [Reco comparison results: 908 differences found in the comparisons](https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_14_1_X_2024-08-04-2300+ea66c8/63879/validateJR.html)
 - DQMHistoTests: Total files compared: 46
 - DQMHistoTests: Total histograms compared: 3524886
 - DQMHistoTests: [Total failures: 5236](https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_14_1_X_2024-08-04-2300+ea66c8/63879/validateJR.html)
 - DQMHistoTests: Total nulls: 2
 - DQMHistoTests: Total successes: 3519628
 - DQMHistoTests: Total skipped: 20
 - DQMHistoTests: Total Missing objects: 0
 - DQMHistoSizes: Histogram memory added: 7.322999999999999 KiB( 45 files compared)
 - DQMHistoSizes: changed ( 23234.0,... ): 1.371 KiB        HLT/Vertexing
 - DQMHistoSizes: changed ( 23234.0,... ): -0.326 KiB       HLT/Tracking
 - DQMHistoSizes: changed ( 29696.0,... ): 0.004 KiB        MessageLogger/Warnings
 - Checked 200 log files, 170 edm output root files, 46 DQM output files
 - [TriggerResults](https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_14_1_X_2024-08-04-2300+ea66c8/63879/triggerResults): no differences found

AuroraPerego commented:
> We can make a commit on top of this PR such that you can just cherry-pick it

Hi @mmusich, I think you can use [this commit](https://github.com/AuroraPerego/cmssw/commit/5039ba71549fc0f5fa828a8b64ff7142ffbaf457). I've checked that the collections are saved in the event content. Let us know if it works :)

mmusich committed:
add renamed tracking at HLT products to phase-2 FEVTDEBUGHLTEventContent

cmsbuild commented:
+code-checks

Logs: https://cmssdt.cern.ch/SDT/code-checks/cms-sw-PR-45636/41161

- There are other open Pull requests which might conflict with changes you have proposed:
  - File  RecoHGCal/Configuration/python/RecoHGCal_EventContent_cff.py  modified in PR(s): #45333

cmsbuild commented:
Pull request #45636 was updated. @Martin-Grunewald, @antoniovagnerini, @antoniovilela, @cmsbuild, @davidlange6, @fabiocos, @jfernan2, @mandrenguyen, @mmusich, @nothingface0, @rappoccio, @rvenditti, @srimanob, @subirsarkar, @syuvivida, @tjavaid can you please check and sign again.

mmusich commented:
@cmsbuild, please test

cmsbuild commented:
+1

**Size**: This PR adds an extra 220KB to repository
**Summary**: https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-ea66c8/40788/summary.html
**COMMIT**: 7d9197171d650566792024298706545d67ff44b0
**CMSSW**: CMSSW_14_1_X_2024-08-05-2300/el8_amd64_gcc12
**User test area**: For local testing, you can use `/cvmfs/cms-ci.cern.ch/week1/cms-sw/cmssw/45636/40788/install.sh` to create a dev area with all the needed externals and cmssw changes.


## Comparison Summary



**Summary**:
 - You potentially [added 88 lines](https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_14_1_X_2024-08-05-2300+ea66c8/63900/validateJR/logRootQA-events.log) to the logs
 - ROOTFileChecks: Some differences in event products or their sizes found
 - [Reco comparison results: 999 differences found in the comparisons](https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_14_1_X_2024-08-05-2300+ea66c8/63900/validateJR.html)
 - DQMHistoTests: Total files compared: 46
 - DQMHistoTests: Total histograms compared: 3524886
 - DQMHistoTests: [Total failures: 2497](https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_14_1_X_2024-08-05-2300+ea66c8/63900/validateJR.html)
 - DQMHistoTests: Total nulls: 2
 - DQMHistoTests: Total successes: 3522367
 - DQMHistoTests: Total skipped: 20
 - DQMHistoTests: Total Missing objects: 0
 - DQMHistoSizes: Histogram memory added: 7.322999999999999 KiB( 45 files compared)
 - DQMHistoSizes: changed ( 23234.0,... ): 1.371 KiB        HLT/Vertexing
 - DQMHistoSizes: changed ( 23234.0,... ): -0.326 KiB       HLT/Tracking
 - DQMHistoSizes: changed ( 29696.0,... ): 0.004 KiB        MessageLogger/Warnings
 - Checked 200 log files, 170 edm output root files, 46 DQM output files
 - [TriggerResults](https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_14_1_X_2024-08-05-2300+ea66c8/63900/triggerResults): no differences found

srimanob commented on Configuration/EventContent/python/EventContent_cff.py:
```diff
@@ -677,8 +677,12 @@ def SwapKeepAndDrop(l):
                             'keep *_hltPhase2PixelTracks_*_*',
-                            'keep *_hltPhase2PixelVertices_*_*'
+                            'keep *_hltPhase2PixelVertices_*_*',
+                            'keep *_hltGeneralTracks_*_*',
```
What is the size increasing with this hltGeneralTracks collection? FEVTDEBUGHLT is super big already, and this will effect the relvals size and Phase-2 production.

mmusich replied:
> What is the size increasing with this hltGeneralTracks collection? 

Zero. This is just a renaming.

mmusich commented:
hold

> DQMHistoSizes: changed ( 23234.0,... ): 1.371 KiB HLT/Vertexing
> DQMHistoSizes: changed ( 23234.0,... ): -0.326 KiB HLT/Tracking

There are residual differences that need to be fixed.

cmsbuild commented:
Pull request has been put on hold by  @mmusich
They need to issue an `unhold` command to remove the `hold` state or L1 can `unhold` it for all

mmusich committed:
rename HGCalRechits for Trigger DQM monitoring

cmsbuild commented:
+code-checks

Logs: https://cmssdt.cern.ch/SDT/code-checks/cms-sw-PR-45636/41177

- There are other open Pull requests which might conflict with changes you have proposed:
  - File  RecoHGCal/Configuration/python/RecoHGCal_EventContent_cff.py  modified in PR(s): #45333

cmsbuild commented:
Pull request #45636 was updated. @Martin-Grunewald, @antoniovagnerini, @antoniovilela, @cmsbuild, @davidlange6, @fabiocos, @jfernan2, @mandrenguyen, @mmusich, @nothingface0, @rappoccio, @rvenditti, @srimanob, @subirsarkar, @syuvivida, @tjavaid can you please check and sign again.

cmsbuild commented:
+code-checks

Logs: https://cmssdt.cern.ch/SDT/code-checks/cms-sw-PR-45636/41178

- There are other open Pull requests which might conflict with changes you have proposed:
  - File  RecoHGCal/Configuration/python/RecoHGCal_EventContent_cff.py  modified in PR(s): #45333

cmsbuild commented:
Pull request #45636 was updated. @Martin-Grunewald, @antoniovagnerini, @antoniovilela, @cmsbuild, @davidlange6, @fabiocos, @jfernan2, @mandrenguyen, @mmusich, @nothingface0, @rappoccio, @rvenditti, @srimanob, @subirsarkar, @syuvivida, @tjavaid can you please check and sign again.

mmusich commented on HLTriggerOffline/Common/python/HLTValidation_cff.py:
```diff
@@ -22,9 +22,9 @@
 from Validation.HGCalValidation.hgcalHitCalibrationDefault_cfi import hgcalHitCalibrationDefault as _hgcalHitCalibrationDefault
 hgcalHitCalibrationHLT = _hgcalHitCalibrationDefault.clone()
 hgcalHitCalibrationHLT.folder = "HGCalHitCalibrationHLT"
```
@cms-sw/hcal-dpg-l2 @rovere, this apparently entered via https://github.com/cms-sw/cmssw/pull/44558. 
I would recommend to use `HLT/HGCalHitCalibration` instead (and not create one more top level folder), but I would defer it to the next PR.

mmusich commented:
unhold

mmusich commented:
@cmsbuild, please test

cmsbuild commented:
+1

**Size**: This PR adds an extra 28KB to repository
**Summary**: https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-ea66c8/40801/summary.html
**COMMIT**: 8bd388e801aeedb69c82c541332a093a1949b565
**CMSSW**: CMSSW_14_1_X_2024-08-06-1100/el8_amd64_gcc12
**User test area**: For local testing, you can use `/cvmfs/cms-ci.cern.ch/week1/cms-sw/cmssw/45636/40801/install.sh` to create a dev area with all the needed externals and cmssw changes.


## Comparison Summary



**Summary**:
 - You potentially [removed 2 lines](https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_14_1_X_2024-08-06-1100+ea66c8/63905/validateJR/logRootQA-events.log) from the logs
 - ROOTFileChecks: Some differences in event products or their sizes found
 - [Reco comparison results: 1028 differences found in the comparisons](https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_14_1_X_2024-08-06-1100+ea66c8/63905/validateJR.html)
 - DQMHistoTests: Total files compared: 46
 - DQMHistoTests: Total histograms compared: 3524886
 - DQMHistoTests: [Total failures: 1872](https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_14_1_X_2024-08-06-1100+ea66c8/63905/validateJR.html)
 - DQMHistoTests: Total nulls: 0
 - DQMHistoTests: Total successes: 3522994
 - DQMHistoTests: Total skipped: 20
 - DQMHistoTests: Total Missing objects: 0
 - DQMHistoSizes: Histogram memory added: 7.3149999999999995 KiB( 45 files compared)
 - DQMHistoSizes: changed ( 23234.0,... ): 1.371 KiB        HLT/Vertexing
 - DQMHistoSizes: changed ( 23234.0,... ): -0.326 KiB       HLT/Tracking
 - Checked 200 log files, 170 edm output root files, 46 DQM output files
 - [TriggerResults](https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_14_1_X_2024-08-06-1100+ea66c8/63905/triggerResults): no differences found

mmusich commented:
> DQMHistoSizes: changed ( 23234.0,... ): 1.371 KiB HLT/Vertexing
DQMHistoSizes: changed ( 23234.0,... ): -0.326 KiB HLT/Tracking

from a quick look, differences look as expected due to the input collection renaming.

mandrenguyen commented:
Comparisons are showing differences seemingly related to the HGCAL. This seems unexpected based on the PR description.

mandrenguyen commented:
+reconstruction

srimanob commented:
> Isn't it due to renaming collections? I.e. old collections will get empty plots. Please point to some more substantial change.

Not sure that I misunderstand your comment, @mmusich. For example, if I am looking on
https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_14_1_X_2024-08-06-1100+ea66c8/63905/validateJR/29634.0_TTbar_14TeV+2026D110/all_HLT_step2/c_CaloTowersSorted_towerMaker__HLT_obj_obj_eta.png
The new one is empty plot while we see the distribution in reference. Or I mis-interpret the plot? Thanks.

mmusich commented:
@srimanob 

> The new one is empty plot while we see the distribution in reference. Or I mis-interpret the plot? Thanks.

you are not misinterpreting the plot, but isn't this the expected behaviour?
`towerMaker` got renamed to `hltTowerMaker`  see [here](https://github.com/cms-sw/cmssw/pull/45636/files#diff-99d5cd80b8e581da0cab1e88f270075136748222601400a0568460b08d47d8afR3), so I think we are expecting to see the entries of `*_towerMaker_*_HLT` to become null (because they are now `*_hltTowerMaker_*_HLT`). Isn't it?

srimanob commented:
Hi @mmusich 

But then should DQM be adjusted to the new collection, e.g. new DQM which shows `*_hltTowerMaker_*_HLT` appears?

mmusich commented:
Hi @srimanob 

> But then should DQM be adjusted to the new collection, e.g. new DQM which shows *_hltTowerMaker_*_HLT appears?

but the plots you are pointing to (e.g [here](https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_14_1_X_2024-08-06-1100+ea66c8/63905/validateJR/29634.0_TTbar_14TeV+2026D110/all_HLT_step2/c_CaloTowersSorted_towerMaker__HLT_obj_obj_eta.png)) are *not* produced by DQM, but rather from the comparison of the event contents done by the cmsbot (IIUC) via [this code](https://github.com/cms-sw/cms-bot/blob/master/comparisons/validate.C). 
If I understand correctly this just dumps whatever collections it finds in the reference file corresponding to the supported types and then it compares what available in the target one. After this PR is merged, the new product names should be automatically picked up. Please @cms-sw/reconstruction-l2 correct me if I am mistaken.

srimanob commented:
+Upgrade

tjavaid commented:
+1

- spurious differences seen from unstable WF `12834.7` (open issue #39803 )

rovere commented on HLTrigger/Configuration/python/HLT_75e33/modules/hltParticleFlowSuperClusterHGCalFromTICLUnseeded_cfi.py:
```diff
@@ -1,15 +1,15 @@
-    PFClusters = cms.InputTag("particleFlowClusterHGCalFromTICLUnseeded"),
-    PFSuperClusterCollectionBarrel = cms.string('particleFlowSuperClusterECALBarrel'),
+    PFClusters = cms.InputTag("hltParticleFlowClusterHGCalFromTICLUnseeded"),
+    PFSuperClusterCollectionBarrel = cms.string('hltParticleFlowSuperClusterECALBarrel'),
```
As far as I can tell, I agree with @waredjeb regarding consistency.
On the other hand, this product (I mean the barrel one created by this HGCal-labelled module) is never consumed. So any name would do w/o crashes.

mmusich replied:
Thank @rovere for the check.

>  On the other hand, this product (I mean the barrel one created by this HGCal-labelled module) is never consumed. So any name would do w/o crashes.

I would propose to go ahead with the PR as is now to avoid re-triggering a lot of signatures and then fix this immediately after, if people agree.

mmusich replied:
Fixed at https://github.com/cms-sw/cmssw/pull/45768. Let me know if you spot other problems.

mmusich commented:
+hlt
   * see caveat at https://github.com/cms-sw/cmssw/pull/45636#discussion_r1703888677 (will fix immediately after this PR is merged in a HLT-only PR).

antoniovilela commented:
+1

cmsbuild commented:
This pull request is fully signed and it will be integrated in one of the next master IBs (tests are also fine). This pull request will be automatically merged.
